src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
-'use strict';
+const logo = document.querySelector('.logo');
 
-// write your code here
+function createResolvedPromise(message) {
+  return new Promise((resolve, reject) => {
+    resolve(message);
+  });
+}
+
+function createRejectedPromise(message, timeout) {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      reject(message);
+    }, timeout);
+  });
+}
+
+function handleSuccess(data) {
+  const successDiv = document.createElement('div');
+
+  successDiv.classList.add('message');
+  successDiv.textContent = data;
+  document.body.appendChild(successDiv);
+}
+
+function handleError(data) {
+  const errorDiv = document.createElement('div');
+
+  errorDiv.classList.add('message', 'error-message');
+  errorDiv.textContent = data;
+  document.body.appendChild(errorDiv);
+}
+
+logo.addEventListener('click', () => {
+  const promise1 = createResolvedPromise('Promise was resolved!');
+
+  promise1.then(handleSuccess).catch(handleError);
+});
+
+const promise2 = createRejectedPromise('Promise was rejected!', 3000);
+
+promise2.then(handleSuccess).catch(handleError);
```

Comment:
The `promise2` is correctly set to reject after a 3000ms timeout. Ensure that the rejection message is handled appropriately by the `handleError` function.